_i18n/es.yml

```diff
@@ -550,7 +550,17 @@ meta:
     description: Las agencias gubernamentales de EE. UU. Pueden aprovechar la plataforma de autenticación y verificación de identidad de login.gov. Obtenga más información sobre las ofertas y oportunidades de login.gov.
     title: Por qué login.gov
 nav:
-  accessiblity: Declaración de accesibilidad
+  accessibility: 
+    title: Accesibilidad
+    sections:
+      - name: Nuestro compromiso
+        url: /accessibility/#nuestro-compromiso
+      - name: Estándares
+        url: /accessibility/#estándares
```

Comment:
Re: Build failure, the `link_to_valid_urls` matcher will not consider this to be a valid URL, since it includes non-ASCII characters (the `á`).

```
$ ruby -e "require 'uri'; URI('/accessibility/#estándares')" 
URI must be ascii only "/accessibility/#est\u00E1ndares" (URI::InvalidURIError)
```

While I think it's technically valid for [modern URL specs](https://url.spec.whatwg.org/), the way I handled this for a similar issue in #510 was to use a substitution for the character. In that case, I used [Jekyll's `slugify: 'ascii'` option](https://jekyllrb.com/docs/liquid/filters/#slugify), though it's not perfect since it would replace anything non-ASCII with a dash `-`. Ideally we could do at least do something instead like [Lodash's `deburr`](https://lodash.com/docs/4.17.15#deburr) to substitute the non-accented form.

Here, since we're doing it manually anyways, we could do this ourselves....

```suggestion
        url: /accessibility/#estandares
```

Either that, or we'll need to update the test logic to be more tolerant to these characters.